alloy/src/main/java/com/liferay/faces/alloy/component/overlay/internal/OverlayRendererBase.java

```diff
@@ -37,48 +37,43 @@
 	protected static final String CONTENT_BOX_SUFFIX = "_contentBox";
 	protected static final String OVERLAY_BODY_SUFFIX = "_overlayBody";
```

Comment:
This file has only minor changes, but since I renamed the `ecodeMarkup` methods to `encodeOverlayMarkup`, jalopy moved everything around.